ext/cl_khr_external_semaphore.asciidoc

```diff
@@ -103,6 +104,13 @@ cl_int clGetSemaphoreHandleForTypeKHR(
     size_t *handle_size_ret);
 ----
 
+----    
+cl_int clImportSemaphoreKHR(
+    cl_semaphore_khr sema_object,
+    cl_semaphore_import_properties_khr *import_props,
+    void *handle_ptr);
```

Comment:
Any specific reason to pass handle_ptr as a separate parameter and not make it one of the props just like supported by clCreateSemaphoreKHR?